README.md

```diff
@@ -1,23 +1,20 @@
-# daal4py - A Convenient Python API to the Intel(R) oneAPI Data Analytics Library
+# scikit-learn-intelex - Extension for scikit-learn with helps Intel(R) oneAPI Data Analytics Library
```

Comment:
what is meant by `with helps`?

Reply:
I mean, that sklearnex using oneDAL backend